skymp5-client/src/view/formView.ts

```diff
@@ -285,6 +285,11 @@ export class FormView implements View<FormModel> {
         if (model.appearance && model.appearance.name) {
           refr?.setDisplayName("" + model.appearance.name, true);
         }
+        if (model.factions !== undefined && model.factions.length > 0) {
+          for (let i = 0; i < model.factions.length; i++) {
+            (Actor.from(refr) as any).addToFaction(Faction.from(Game.getFormEx(model.factions[i].formDesc)));
```

Comment:
Actor.from is expensive. should be put out of loop. also it can return null which should be checked